roles/pgbackrest_srv/templates/pgbackrest.conf.j2

```diff
@@ -0,0 +1,13 @@
+[global]
```

Comment:
please use roles/pgbackrest

it can be a separate template, for example `pgbackrest.server.conf.j2`